.github/workflows/linux.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pytest
+        pip install pytest six
```

Comment:
probably should add resource here, right?

Reply:
`resource` is a python built-in: https://docs.python.org/3/library/resource.html .  It's just unix-only, which is why it has to be optionally-imported to avoid breaking the Windows build.